bitnami/sonarqube/values.yaml

```diff
@@ -578,6 +586,112 @@ ingress:
   ##
   extraRules: []
 
+## @section SonarQube caCerts provisioning parameters
+##
+## Provide a secret containing one or more certificate files in the keys that will be added to cacerts
+## The cacerts file will be set via SONARQUBE_WEB_JVM_OPTS and SONAR_CE_JAVAOPTS
+caCerts:
+  ## @param caCerts.enabled Enable the use of caCerts
+  enabled: true
+  ## Bitnami Shell image
+  ## ref: https://hub.docker.com/r/bitnami/bitnami-shell/tags/
+  ## @param caCerts.image.registry Bitnami Shell image registry
+  ## @param caCerts.image.repository Bitnami Shell image repository
+  ## @param caCerts.image.tag Bitnami Shell image tag (immutable tags are recommended)
+  ## @param caCerts.image.digest Bitnami Shell image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+  ## @param caCerts.image.pullPolicy Bitnami Shell image pull policy
+  ## @param caCerts.image.pullSecrets Bitnami Shell image pull secrets
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/bitnami-shell
+    tag: 11-debian-11-r63
```

Comment:
Please note you need to define this container image in our internal test & release pipeline so the tag is automatically updated as the other images of this Helm chart